components/31-molecules/checkboxes/README.md

```diff
@@ -0,0 +1,26 @@
+# Checkboxes
+
+## When to use this component
+
+Use the checkboxes component to help users:
+
+* Select one or more options from a list a short list of options.
+* Toggle a single option on or off.
+
+## When not to use this component
+
+Do not use the checkboxes component when you need to help users select one or more options from a long list of options. In this cases, use <a href="{{path './checkboxes-with-filter.html'}}">checkboxes with filter component</a> instead.
+
+Do not use the checkboxes component when users should only be able to select one option and no more. In this case, use the <a href="{{path './radios.html'}}">radios component</a> instead.
+
+## How it works
+
+* Checkboxes are grouped in a fieldset.
+* The fieldset uses a legend to present a description for the checkboxes. This is typically a question.
+* Users should be able to select one or more options.
+* Do not select checkboxes by default as users might not notice this, resulting in not realising they missed a question and/or sending a wrong answer.
```

Comment:
@hstandaert What do you think, does this point fit better under "How it works" or under "Usage"?

Reply:
Looking at all other components that have a "Usage" section, I see that it mostly contains implementation details for developers. So I'd say: just leave it here since this is rather a general guideline than a note on how to implement the component.

Reply:
That sounds perfect!